test/langtools/tools/javac/generics/inference/NestedCapture.java

```diff
@@ -26,7 +26,7 @@
  * @bug 8039214
  * @summary Capture variable passed through multiple levels of nested inference
  * @compile NestedCapture.java
- * @compile -Xlint:-options -source 7 NestedCapture.java
+ * @compile NestedCapture.java
```

Comment:
Did too much get removed here?   Should there be a `--release 7`?

Reply:
Good catch; fixed.